### DISCUSSION TITLE
Community Discord server

### DISCUSSION THREAD

**Discussion by rwxzig** [OrchardCMS/OrchardCore#14344] [General]
Hello everyone, 
I created a Discord server for OrchardCMS and anyone can join through [this](https://discord.gg/haceAaBt) link.
If there are any legal implications on the name or something else, please let me know and I would be happy to change in a way that wont conflict.

**Comment by Piedone** [comment]
Great initiative, but there's actually one already :) https://github.com/OrchardCMS/OrchardCore/issues/13707.

**Reply by rwxzig** [reply]
I searched first and did not found, but nonetheless I am **deleting** the server that I created.
